OpenRA.Game/World.cs

```diff
@@ -36,6 +36,8 @@ public sealed class World : IDisposable
 
 		readonly Queue<Action<World>> frameEndActions = new Queue<Action<World>>();
 
+		public readonly int OriginalTimestep;
```

Comment:
I've been struggling to mentally map the modifier to the other places it would affect (there's a few more, most notably `ReplayControlBarLogic`) plus how to avoid rounding down to zero unless explicitly wanted.
For me it was easier to just rename `OriginalTimestep` to `Timestep` and `Timestep` to `ReplayTimestep` and ensure the latter is exclusively used for that. Hope that's good enough.